doc/rm/register_tool/index.md

```diff
@@ -576,6 +576,8 @@ The general behavior of updating a shadow register is as follows:
    The phase is tracked internally to the module.
    If needed, software can put the shadow register back into the first phase by performing a read operation.
    If the register is of type `RO`, software cannot interfere with the update process and read operations do not clear the phase tracker.
+   If the register is of type `RW1C` or `RW0C`, the staged register will latch the raw data value coming from the bus side.
+   The raw data value of the second write is compared with the raw data value in the staged register before being filtered by the `RW1C` or `RW0C` set/clear logic.
```

Comment:
I have also added this to the description of the staged reg. Does that make sense?

Reply:
CC @vogelpi

Reply:
This looks good but is it possible that you mixed up `RW1C` and `RW1S` here and in the code?

Reply:
I am not sure if all three cases `RW1C`, `RW0C` and `RW1S` need special treatment.

Reply:
Thanks for catching this. I mixed up `RW1C` with `RW1S` in the documentation here - let me fix that. 

`RW1C` is currently not supported by the shadow reg and hence not mentioned here (there is a check for this in the Python scripting that will throw an error).